app/controllers/explore_controller.rb

```diff
@@ -2,11 +2,14 @@
 
 class ExploreController < ApplicationController
   def index
-    @platforms = Project.maintained.group(:platform).order("count_id DESC").limit(28).count("id").keys
-    @languages = Project.maintained.where.not(language: nil).group(:language).order("count_id DESC").limit(21).count("id").keys
-
-    project_scope = Project.includes(:repository).maintained.with_description
+    @platforms = Rails.cache.fetch("explore:platforms", expires_in: 1.day, race_condition_ttl: 2.minutes) do
+      Project.maintained.group(:platform).order("count_id DESC").limit(28).count("id").keys
+    end
+    @languages = Rails.cache.fetch("explore:languages", expires_in: 1.day, race_condition_ttl: 2.minutes) do
+      Project.maintained.where.not(language: nil).group(:language).order("count_id DESC").limit(21).count("id").keys
+    end
 
+    project_scope = Project.maintained.with_description
```

Comment:
This used to also include all the repositories for the `Project` but I don't see any usage of `Repository` fields in the query.